introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/inference-benchmarking-container-example.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        # },
</pre><p>Should we remove the commented code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>